package.json

```diff
@@ -17,7 +17,9 @@
     "css:stats": "analyze-css --file build/bitstyles.css --pretty > test/css-stats.json",
     "precss:graph": "yarn build",
     "css:graph": "specificity-graph build/bitstyles.css --output test/specificity_graph/",
-    "build-storybook": "build-storybook"
+    "build-storybook": "yarn build && build-storybook",
+    "predeploy-storybook": "yarn build-storybook",
```

Comment:
How is this command used? Is it automatically triggered by some internal magic of the `gh-pages` package?

If not, I wonder if it would not make sense to integrate it with the `deploy-storybook` command to ensure it's never forgotten.

Reply:
It’s magic, but from yarn/npm :D if you prepend `pre` to the name of an existing script, (`deploy-storybook` in this case, which is on the next line) yarn/npm will run that script before running the script itself when you call `yarn script-name`.

Reply:
🤯 wow, thanks 🙇